pkg/operation/botanist/worker.go

```diff
@@ -45,6 +45,7 @@ func (b *Botanist) DefaultWorker() worker.Interface {
 			Region:            b.Shoot.GetInfo().Spec.Region,
 			Workers:           b.Shoot.GetInfo().Spec.Provider.Workers,
 			KubernetesVersion: b.Shoot.KubernetesVersion,
+			CloudProfile: b.Shoot.CloudProfile,
```

Comment:
Not formatted. 
Please run `make format` and `make verify` locally before pushing.